docs/integrations/duckdb.md

```diff
@@ -26,17 +26,15 @@ Then run the following to configure the connection.
 
 ```sql
 SET s3_region='us-east-1';
-SET s3_endpoint='lakefs.example.com';
-SET s3_access_key_id='AKIAIOSFODNN7EXAMPLE';
-SET s3_secret_access_key='wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY';
+SET s3_endpoint='lakefs.example.com'; -- the host (and port, if necessary) of your lakeFS server
+SET s3_access_key_id='AKIAIOSFODNN7EXAMPLE'; -- the access credentials for your lakeFS user
+SET s3_secret_access_key='wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY'; -- the access credentials for your lakeFS user
 SET s3_url_style='path';
 
 -- Uncomment in case the endpoint listen on non-secure, for example running lakeFS locally.
 -- SET s3_use_ssl=false;
 ```
 
-* `s3_endpoint` is the host (and port, if necessary) of your lakeFS server
-* `s3_access_key_id` and `s3_secret_access_key` are the access credentials for your lakeFS user
```

Comment:
all or nothing? suggest we keep the legend or have all the field's descriptions inside the snippet.

Reply:
done, I removed the line about path style, which seems redundant